internal/datastore/mysql/migrations/driver.go

```diff
@@ -0,0 +1,124 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strings"
+
+	sq "github.com/Masterminds/squirrel"
+	sqlDriver "github.com/go-sql-driver/mysql"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	errUnableToInstantiate       = "unable to instantiate MysqlDriver: %w"
+	mysqlMissingTableErrorNumber = 1146
+
+	migrationVersionColumnPrefix = "_meta_version_"
+)
+
+var sb = sq.StatementBuilder.PlaceholderFormat(sq.Question)
+
+type MysqlDriver struct {
+	db *sql.DB
+	*tables
+}
+
+// NewMysqlDriverFromDSN creates a new migration driver with a connection pool to the database DSN specified.
+//
+// URI: [scheme://][user[:[password]]@]host[:port][/schema][?attribute1=value1&attribute2=value2...
+// See // https://dev.mysql.com/doc/refman/8.0/en/connecting-using-uri-or-key-value-pairs.html
```

Comment:
drop `//` after the `See`

Reply:
addressed in https://github.com/authzed/spicedb/pull/525/commits/8e7fb6ec22b2df5f3bd5f6d0c268c86a8bdb92f8